### ISSUE TITLE
Saving argument settings with name

### ISSUE THREAD

**Issue by clemera** [magit/transient#91]
I think it would be nice to have a way to save a set arguments of a transient under a user specified name. Then later one could select such a named combo via completion.

**Comment by tarsius** [comment]
Sorry, I ignored this for a while.
And I am afraid I will continue to do so because I am on vacation.

**Comment by clemera** [comment]
Oh, no problem but this sounds like you are open to the idea at least :) I think this feature would have similar utility as the alias feature in shells and with transient you could create an alias on the fly.

**Comment by tarsius** [comment]
After thinking about it a bit, I am not so sure this would actually be useful.

Why don't you implement this yourself and try it out for a while, and then report back here?

It should be fairly simple to do so. Basically you have to copy-paste `transient-save`, `transient-save-value`, `transient-save-values`, `transient-values` and `transient--pp-to-file`, creating variants that save "aliases". In addition to `transient-save-alias`, you might want to create `transient-delete-alias` as well.

**Comment by clemera** [comment]
Thanks for the pointers I will give it a try next time I start messing with my transient config ;) I think the idea has potential, for example when using find or rg I would like to save common flags I use to search for certain things.

**Comment by tarsius** [comment]
Right. I didn't actually mean to say that it wouldn't be useful at all. I just feel that a minimal viable version wouldn't be that useful and that it would be quite a bit of work to figure out what else is needed and then also implement it.

I remain open to the idea of maybe adding this in the future but could also see it as a separate package.
